src/display/annotation_layer.js

```diff
@@ -244,7 +244,8 @@ class AnnotationElement {
           break;
       }
 
-      if (data.color) {
+      const borderColor = data.borderColor || data.color || null;
+      if (borderColor) {
         container.style.borderColor = Util.makeHexColor(
           data.color[0] | 0,
           data.color[1] | 0,
```

Comment:
Is this `makeHexColor`-call *supposed* to use `data.color` now, since naively I'd assume that it should actually use `borderColor` instead?
As-is the borderColor is being ignored, which I'm assuming wasn't intended; was this perhaps fallout from addressing one of the review comments?

Reply:
`borderColor` is set from `MK-BC` entry and `color` is set from `C` one.
If I understand the spec correctly, `BC` (see https://www.adobe.com/content/dam/acom/en/devnet/pdf/pdfs/PDF32000_2008.pdf#page=417) is optional and for widget annotations, when `C` (see https://www.adobe.com/content/dam/acom/en/devnet/pdf/pdfs/PDF32000_2008.pdf#page=392) is for icon, popup and link, so only one of the `BC` and `C` should be set.

Reply:
If I'm correct, I should likely add a comment.